src/docs/catalog/samples/tabbed-app-bar.md

```diff
@@ -4,7 +4,7 @@ description: An example that uses a TabBar for its AppBar.
 deprecated: true
 ---
 
-An AppBar with a TabBar as its bottom widget.
+ Sử dụng AppBar với TabBar ở bên dưới widget.
```

Comment:
You don't have to translate words one-by-one. Make sure everyone easy to understand.
Something like "Một Appbar với TabBar ở phía dưới cùng." is ok.